cmd/serve/httplib/httplib.go

```diff
@@ -201,6 +203,19 @@ func NewServer(handler http.Handler, opt *Options) *Server {
 		}
 		oldHandler := handler
 		handler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			// No auth if the request is for the web-gui
+			requestURI := strings.Split(r.RequestURI, "/")
+			fs.Debugf(nil, "RequestURI: %s, %d", r.RequestURI, len(requestURI))
+			// No auth for serving web UI files
+			if s.Opt.WebUI && r.Method == "GET" && (len(requestURI) == 2 || (len(requestURI) > 1 && requestURI[1] == "static")) {
```

Comment:
I don't quite understand the logic here... It seems to say if `r.RequestURL` has two `/` then it is OK or if it has more than 1 slash and starts with `/static/` then it is OK.

That doesn't seem quite right!

Can you explain in words what logic you wanted?

Reply:
I wanted to permit the access to Root and anything that starts with static/**

Reply:
I see - I forgot that "/" will split into two parts!  I think something like this would be a lot clearer.

```go
if s.Opt.WebUI && r.Method == "GET" && (r.RequestURI == "/" || strings.HasPrefix(r.RequestURI, "/static/") {
```

Reply:
Actually, r.requestURI == "/" wont work, because there are many files in the root as well.
I will try using regex

Reply:
Done :+1: